### ISSUE TITLE
Add Sitemap file to docs

### ISSUE THREAD

**Issue by MaedahBatool** [sourcegraph/docs#360]
Right now our docs stack has no sitemap file. We need to add one.

**Comment by MaedahBatool** [comment]
@zahidalidev While we wait for the Changelog, let's start working on adding a Sitemap if you have some time.

**Comment by zahidalidev** [comment]
Hi Maedah,

I hope you're doing well. Please review the PR I submitted for the Tabs
component. I've included a detailed description explaining how to use it: PR
#402 <https://github.com/sourcegraph/docs/pull/402#issue-2344554630>.

Once this PR is merged, I will create another PR addressing the tech stack
update and hydration warnings. Meanwhile, I will also look into the Sitemap.

Thank you.

**Comment by MaedahBatool** [comment]
All your previous PRs are merged I believe we are good to go

**Comment by zahidalidev** [comment]
Yes, Thank you so much. I will try to complete the sitemap task before our
meeting tomorrow.
